src/CodeInjector.ts

```diff
@@ -486,6 +486,24 @@ canvas = vcs.init(display_target='off')`;
     );
   }
 
+  /**
+   * Updates the colormap thats to be used by the selected graphics method
+   * @param gm_name: the name of the graphics method to update the color map for
+   * @param cm_name: the name of the color map
+   * @returns Promise<[number, string]> - A promise for when the cell code has executed containing
+   * the cell's index and output result
+   */
+  public async updateColormapName(gm_name: string, gm_group: string, cm_name: string): Promise<[number, string]>{
+    let cmd = `${gm_name}_${gm_group}.colormap = '${cm_name}'`;
```

Comment:
There is some kind of issue going on with the graphic method and colormap code injection. I tried selecting a colormap but it gives an error due to the graphic method not being initialized. I was trying to look where the issue is but it doesn't seem straight forward, so let me know if you'd like me to troubleshoot it?

Reply:
I think I found out the issue and will have a fix for it.

Reply:
Minor styling issue: The variable names should match the camel case used by other variables in the file, not sure why this passed the linter